src/scipp/plotting/profile.py

```diff
@@ -126,3 +126,12 @@ def toggle_norm(self, owner):
         """
         self.ax.set_yscale("log" if owner.value else "linear")
         self.draw()
+
+    def rescale_to_data(self, button=None):
+        """
+        Rescale y axis to the contents of the plot.
+        """
+        self.ax.autoscale(True)
```

Comment:
Does this need to be reset at the end of the function?

Reply:
In the end I just removed it, I dont think we need it.